core/tests/wdio.conf.js

```diff
@@ -302,10 +302,20 @@ exports.config = {
         console.log(`ffmpeg exited with code ${code}`);
       });
     }
-    // Set a wide enough window size for the navbar in the library pages to
-    // display fully.
-    browser.setWindowSize(1285, 1000);
-
+    // If the tests are running in debug mode,
+    // the mobile viewport information is passed as
+    // the 9th argument, else it is passed as the 8th argument.
```

Comment:
This seems really fragile. Is there another way to pass this information, for example with an environment variable? Alternatively, are there better ways to handle arguments so that we don't have to do this ad-hoc parsing? ditto for protractor

Reply:
Done. I'm using env variables now.